gatsby-config.ts

```diff
@@ -23,12 +25,13 @@ const config: GatsbyConfig = {
     {
       resolve: `gatsby-plugin-manifest`,
       options: {
-        name: "DIAL UP DIGITAL",
-        short_name: "DIAL UP DIGITAL",
+        name: isAppsSite ? "Nota" : "DIAL UP DIGITAL",
+        short_name: isAppsSite ? "Nota" : "DIAL UP DIGITAL",
         start_url: ".",
         background_color: "#ffffff",
         theme_color: "#000000",
         display: 'minimal-ui',
+        // TODO: Update so that deploys of the DUD portfolio site use the original DUD favicon.
```

Comment:
We can probably fix this by making a small edit upon #64 to have to favicon files, *_dud.* and *_nota.*, that we select here conditionally depending on `isAppsSite`